log_analyzer/src/templates/base.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/3.1.2/socket.io.js"
+            integrity="sha512-YybopSVjZU0fe8TY4YDuQbP5bhwpGBE/T6eBUEZ0usM72IWBfWrgVI13qfX4V2A/W7Hdqnm7PIOYOwP9YHnICw==
+            " crossorigin="anonymous"></script>
+    <!--  Bootstrap start  -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
```

Comment:
What if there is no internet? Would these work?

Reply:
No, it won't. Is that a bad way, should I download these libraries?

Reply:
If it was a live internet site, I would surely say yes. The main reason that the visitor does not want to be tracked by the server hosting these libs. In our case, we are not doing a live site, but the tracking will still be totally possible. So I would say, don't user these links.
But what I would still do is to make them part of the build. So instead of downloading and putting them into the repo, I would do a build script which downloads the dependencies into a folder which is git ignored. And that is it. So basically the starting script can make sure that he environment is set up and that's it.